### ISSUE TITLE
Feature request: support git commit --amend

### ISSUE THREAD

**Issue by alexanderkogan** [initialcommit-com/git-sim#33]
Currently only `git commit` is supported.
Amending would be quite easy to show, I think.

When I find the time, I might try to implement it, but if anybody feels the urge, feel free to do it. (;
From my first foray through the code, it would be similar to the additional flags in `reset`.

**Comment by initialcommit-io** [comment]
@alexanderkogan Yes this should be a quick one I'll try to get it implemented and released later today!  I'll post back here when ready for you to test.

**Comment by initialcommit-io** [comment]
@alexanderkogan Just implemented this in git-sim 0.1.7. You can now use the `--amend` option as a part of git-sim commit as follows:

```
$ git-sim commit --amend -m "Amended commit message"
```

Note that for now the amended commit message must be supplied using the `-m` flag.

Please upgrade to git-sim 0.1.7 by running:

```
$ pip install git-sim --upgrade
```

Then test it out and let me know if it works for you! =D

**Comment by initialcommit-io** [comment]
@alexanderkogan Closing since this is implemented now.  Feel free to reopen if any issues.
